src/components/WalletCompare.js

```diff
@@ -501,7 +501,7 @@ const WalletCompare = ({ location }) => {
           <p>
             <em>
               <Translation id="page-find-wallet-not-endorsements" />{" "}
-              <Link to="/en/contributing/adding-products/">
```

Comment:
This hard-coding of `/en` is intentional here since the contributing page is not translated.

The same is true for any of the pages flagged as `isLegal` in `gatsby-node`:
https://github.com/ethereum/ethereum-org-website/blob/0f71a8d2d6beae0c02224310c29bddf594b9921e/gatsby-node.js#L137
```
const isLegal =
      slug.includes(`/cookie-policy/`) ||
      slug.includes(`/privacy-policy/`) ||
      slug.includes(`/terms-of-use/`) ||
      slug.includes(`/contributing/`)
```